.github/workflows/e2e.yml

```diff
@@ -75,7 +75,7 @@ jobs:
           TESTFLAGS: -coverprofile=/tmp/coverage/coverage.txt
       -
         name: Send to Codecov
-        uses: codecov/codecov-action@v4
+        uses: codecov/codecov-action@v5
         with:
           file: ./build/coverage/coverage.txt
```

Comment:
Looks like this needs an update here; https://github.com/codecov/codecov-action/releases/tag/v5.0.0

<img width="1129" alt="Screenshot 2024-11-15 at 11 22 58" src="https://github.com/user-attachments/assets/235f8b99-05b3-4f9f-8f10-01f63dcfd285">